src/solver/conv_asm_3x3u.cpp

```diff
@@ -70,8 +70,7 @@ PerformanceConfigConvAsm3x3U::PerformanceConfigConvAsm3x3U(int lwc, int fpw, int
 {
 }
 
-inline bool
-PerformanceConfigConvAsm3x3U::operator==(const PerformanceConfigConvAsm3x3U& other) const
+bool PerformanceConfigConvAsm3x3U::operator==(const PerformanceConfigConvAsm3x3U& other) const
```

Comment:
Can we move the inlined operators to the headers ? That would keep them inline and avoid a function call cost?

Reply:
Sure, I'll move those around.

Reply:
This is ONLY called from generic search. It would not have ANY noticeable difference.